app/main.py

```diff
@@ -1 +1,16 @@
-# write your code here
+def copy_file(command: str) -> None:
+    if len(command.split()) == 3:
+        command_name = command.split()[0]
+        source_file_name = command.split()[1]
+        destination_file_name = command.split()[2]
```

Comment:
Avoid performing `split` operation for 3 times here, use unpacking instead
```suggestion
        command_name, source_file_name, destination_file_name = command.split()
```